src/watcher/watcher.ts

```diff
@@ -7,6 +7,12 @@ import { ethers } from "ethers";
 import { EthereumResponderManager } from "../responder";
 import { ReorgDetector } from "../blockMonitor";
 import { inspect } from "util";
+import levelup, { LevelUp } from "levelup";
+import leveldown from "leveldown";
+import encodingDown from "encoding-down";
+import EncodingDown from "encoding-down";
+import { Readable } from "stream";
+import { ReadStream, createReadStream } from "fs";
 
```

Comment:
All these imports are not used in this file.

Reply:
removed